### PR TITLE
KRACOEUS-8912: Add ability to certify enroute proposal

### PR DESCRIPTION
**Pull request by gathreya**

### CODE REVIEW

File: coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java [hunk 1]
```diff
@@ -55,6 +57,7 @@
 import org.kuali.coeus.propdev.impl.questionnaire.ProposalDevelopmentQuestionnaireHelper;
 import org.kuali.coeus.propdev.impl.s2s.question.ProposalDevelopmentS2sQuestionnaireHelper;
 import org.kuali.coeus.sys.framework.controller.KcFileService;
+import org.kuali.coeus.sys.framework.service.KcServiceLocator;
```
Comment (jwillia) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: unused import

Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: indeed :)

**Comment by gathreya** [comment]
fixed

File: coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java [hunk 2]
```diff
@@ -763,6 +770,18 @@ protected boolean renderEditForPersonnelAttachment(String personId, DevelopmentP
         return true;
     }
 
+    public boolean renderSaveForCertification(ProposalDevelopmentDocument document) {
```
Comment (jwillia) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: can all this code go in the docAuthorizer and use it to set an action flag or edit mode there.  Then use the action flag or edit mode to render the save button?

Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: yeah, I actually am working on that now. I realized I could just do that after I created this PR. I will update soon

Comment (gathreya) [inline coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java]: there's a couple of other things I need to fix in this PR as well. WIll update soon

**Comment by gathreya** [comment]
ok, here it is

**Pull request merged**